.github/memory_statistics_config.json

```diff
@@ -2,10 +2,13 @@
     "lib_name": "AWS IoT Jobs",
     "src": [
         "source/jobs.c",
-        "coreJSON/source/core_json.c"
+        "coreJSON/source/core_json.c",
```

Comment:
Can you swap this with line 6+7 below? I'd prefer to keep all the source items together.

Reply:
Sure!